src/pal/src/exception/signal.cpp

```diff
@@ -246,8 +246,11 @@ BOOL SEHInitializeSignals(DWORD flags)
     handle_signal(SIGBUS, sigbus_handler, &g_previous_sigbus);
     // SIGSEGV handler runs on a separate stack so that we can handle stack overflow
     handle_signal(SIGSEGV, sigsegv_handler, &g_previous_sigsegv, SA_ONSTACK);
-    handle_signal(SIGINT, sigint_handler, &g_previous_sigint);
-    handle_signal(SIGQUIT, sigquit_handler, &g_previous_sigquit);
+    // We don't setup a handler for SIGINT/SIGQUIT when those signals are ignored.
+    // Otherwise our child processes would reset to the default on exec causing them
+    // to terminate on these signals.
+    handle_signal(SIGINT, sigint_handler, &g_previous_sigint, 0, 1);
```

Comment:
Could you please add inline comment with names for each of the last two parameters? like 
```C++
handle_signal(SIGINT, sigint_handler, &g_previous_sigint, 0 /* additionalFlags */ , 1 /* skipIgnored */);
```